### PR TITLE
blah blah adds some initialize() for squeaky clean code and performance

### PR DESCRIPTION
**Pull request by praisenarsie**

### CODE REVIEW

File: code/modules/food_and_drinks/kitchen_machinery/juicer.dm [hunk 1]
```diff
@@ -166,22 +166,22 @@
...
-	new/obj/item/reagent_containers/food/snacks/grown/berries(src)
-	new/obj/item/reagent_containers/food/snacks/grown/banana(src)
-	new/obj/item/reagent_containers/food/snacks/grown/grapes(src)
-	new/obj/item/reagent_containers/food/snacks/grown/tomato(src)
-	new/obj/item/reagent_containers/food/snacks/grown/carrot(src)
-	new/obj/item/reagent_containers/food/snacks/grown/berries(src)
-	new/obj/item/reagent_containers/food/snacks/grown/banana(src)
-	new/obj/item/reagent_containers/food/snacks/grown/grapes(src)
+/obj/structure/closet/crate/juice/Initialize()
+	. = ..()
```
Comment (Cyberboss) [inline code/modules/food_and_drinks/kitchen_machinery/juicer.dm]: This should be PopulateContents and just ..()

File: code/modules/food_and_drinks/pizzabox.dm [hunk 2]
```diff
@@ -30,9 +30,9 @@
 	var/const/BOMB_TIMER_MIN = 1
 	var/const/BOMB_TIMER_MAX = 10
 
-/obj/item/pizzabox/New()
+/obj/item/pizzabox/Initialize()
 	update_icon()
-	..()
+	. = ..()
```
Comment (Cyberboss) [inline code/modules/food_and_drinks/pizzabox.dm]: . = ..() first

File: code/modules/food_and_drinks/pizzabox.dm [hunk 3]
```diff
@@ -259,29 +259,29 @@
...
 	update_icon()
 
-/obj/item/pizzabox/bomb/New()
+/obj/item/pizzabox/bomb/Initialize()
 	var/randompizza = pick(subtypesof(/obj/item/reagent_containers/food/snacks/pizza))
 	pizza = new randompizza(src)
 	bomb = new(src)
 	wires = new /datum/wires/explosive/pizza(src)
-	..()
+	. = ..()
```
Comment (Cyberboss) [inline code/modules/food_and_drinks/pizzabox.dm]: . = ..() first

File: code/modules/food_and_drinks/pizzabox.dm [hunk 4]
```diff
@@ -259,29 +259,29 @@
...
 
-/obj/item/pizzabox/margherita/New()
+/obj/item/pizzabox/margherita/Initialize()
 	pizza = new /obj/item/reagent_containers/food/snacks/pizza/margherita(src)
 	boxtag = "Margherita Deluxe"
-	..()
+	. = ..()
```
Comment (Cyberboss) [inline code/modules/food_and_drinks/pizzabox.dm]: . = ..() first

File: code/modules/food_and_drinks/pizzabox.dm [hunk 5]
```diff
@@ -259,29 +259,29 @@
...
 
-/obj/item/pizzabox/vegetable/New()
+/obj/item/pizzabox/vegetable/Initialize()
 	pizza = new /obj/item/reagent_containers/food/snacks/pizza/vegetable(src)
 	boxtag = "Gourmet Vegatable"
-	..()
+	. = ..()
```
Comment (Cyberboss) [inline code/modules/food_and_drinks/pizzabox.dm]: . = ..() first

File: code/modules/food_and_drinks/pizzabox.dm [hunk 6]
```diff
@@ -259,29 +259,29 @@
...
 
-/obj/item/pizzabox/mushroom/New()
+/obj/item/pizzabox/mushroom/Initialize()
 	pizza = new /obj/item/reagent_containers/food/snacks/pizza/mushroom(src)
 	boxtag = "Mushroom Special"
-	..()
+	. = ..()
```
Comment (Cyberboss) [inline code/modules/food_and_drinks/pizzabox.dm]: . = ..() first

File: code/modules/food_and_drinks/pizzabox.dm [hunk 7]
```diff
@@ -259,29 +259,29 @@
...
 
-/obj/item/pizzabox/meat/New()
+/obj/item/pizzabox/meat/Initialize()
 	pizza = new /obj/item/reagent_containers/food/snacks/pizza/meat(src)
 	boxtag = "Meatlover's Supreme"
-	..()
+	. = ..()
```
Comment (Cyberboss) [inline code/modules/food_and_drinks/pizzabox.dm]: . = ..() first

File: code/modules/holodeck/items.dm [hunk 8]
```diff
@@ -42,9 +42,9 @@
 /obj/item/holo/esword/attack(target as mob, mob/user as mob)
 	..()
 
-/obj/item/holo/esword/New()
+/obj/item/holo/esword/Initialize()
 	item_color = pick("red","blue","green","purple")
-	..()
+	. = ..()
```
Comment (Cyberboss) [inline code/modules/holodeck/items.dm]: . = ..() first

File: code/modules/mob/living/carbon/alien/organs.dm [hunk 9]
```diff
@@ -3,12 +3,12 @@
...
 	var/list/alien_powers = list()
 
-/obj/item/organ/alien/New()
+/obj/item/organ/alien/Initialize()
 	for(var/A in alien_powers)
 		if(ispath(A))
 			alien_powers -= A
 			alien_powers += new A(src)
-	..()
+	. = ..()
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/alien/organs.dm]: . = ..() first

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/gutlunch.dm [hunk 10]
```diff
@@ -123,7 +123,7 @@
 /obj/item/udder/gutlunch
 	name = "nutrient sac"
 
-/obj/item/udder/gutlunch/New()
+/obj/item/udder/gutlunch/Initialize()
 	reagents = new(50)
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/gutlunch.dm]: Add . = ..()

File: code/modules/modular_computers/hardware/hard_drive.dm [hunk 11]
```diff
@@ -121,9 +121,9 @@
 	stored_files = null
 	return ..()
 
-/obj/item/computer_hardware/hard_drive/New()
+/obj/item/computer_hardware/hard_drive/Initialize()
 	install_default_programs()
-	..()
+	. = ..()
```
Comment (Cyberboss) [inline code/modules/modular_computers/hardware/hard_drive.dm]: . = ..() first

File: code/modules/paperwork/filingcabinet.dm [hunk 12]
```diff
@@ -185,7 +185,7 @@ GLOBAL_LIST_EMPTY(employmentCabinets)
 	icon_state = "employmentcabinet"
 	var/virgin = 1
 
-/obj/structure/filingcabinet/employment/New()
+/obj/structure/filingcabinet/employment/Initialize()
 	GLOB.employmentCabinets += src
 	return ..()
```
Comment (Cyberboss) [inline code/modules/paperwork/filingcabinet.dm]: . = ..() first

File: code/modules/paperwork/pen.dm [hunk 13]
```diff
@@ -181,12 +181,12 @@
...
 
 
-/obj/item/pen/sleepy/New()
+/obj/item/pen/sleepy/Initialize()
 	create_reagents(45)
 	reagents.add_reagent("chloralhydrate2", 20)
 	reagents.add_reagent("mutetoxin", 15)
 	reagents.add_reagent("tirizene", 10)
-	..()
+	. = ..()
```
Comment (Cyberboss) [inline code/modules/paperwork/pen.dm]: . = ..() first

File: code/modules/power/lighting.dm [hunk 14]
```diff
@@ -193,23 +193,23 @@
 /obj/machinery/light/built
 	icon_state = "tube-empty"
 
-/obj/machinery/light/built/New()
+/obj/machinery/light/built/Initialize()
 	status = LIGHT_EMPTY
 	update(0)
-	..()
+	. = ..()
```
Comment (Cyberboss) [inline code/modules/power/lighting.dm]: . = ..() first

File: code/modules/power/lighting.dm [hunk 15]
```diff
@@ -193,23 +193,23 @@
...
 
 /obj/machinery/light/small/built
 	icon_state = "bulb-empty"
 
-/obj/machinery/light/small/built/New()
+/obj/machinery/light/small/built/Initialize()
 	status = LIGHT_EMPTY
 	update(0)
-	..()
+	. = ..()
```
Comment (Cyberboss) [inline code/modules/power/lighting.dm]: . = ..() first

File: code/modules/hydroponics/grown/random.dm [hunk 16]
```diff
@@ -12,9 +12,9 @@
 	icon_harvest = "xpod-harvest"
 	growthstages = 4
 
-/obj/item/seeds/random/New()
+/obj/item/seeds/random/Initialize()
 	randomize_stats()
-	..()
+	. = ..()
```
Comment (Cyberboss) [inline code/modules/hydroponics/grown/random.dm]: . = ..() first

Comment (Cyberboss) [reply code/modules/hydroponics/grown/random.dm]: Also you need to port seeds to Initialize or this will break things

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/games/cas.dm [hunk 17]
```diff
@@ -27,7 +27,7 @@
 	decksize = 50
 	card_text_file = "strings/cas_black.txt"
 
-/obj/item/toy/cards/deck/cas/New()
+/obj/item/toy/cards/deck/cas/Initialize()
```
Comment (praisenarsie) [inline code/modules/games/cas.dm]: does this need . = ..() first?

Comment (ShizCalev) [reply code/modules/games/cas.dm]: Current location should be fine.

**Review by Cyberboss — APPROVED** [review]

**Comment by Cyberboss** [comment]
Travis did finish, but it's being a shitter

**Pull request merged**